### PR TITLE
update: based on spec, updated messages of notation key command

### PR DESCRIPTION
**Pull request by Two-Hearts**
This PR is based on spec changes in here: https://github.com/notaryproject/notation/pull/606.

### CODE REVIEW

File: cmd/notation/key.go [hunk 1]
```diff
@@ -243,10 +240,11 @@ func deleteKeys(ctx context.Context, opts *keyDeleteOpts) error {
 	}
 
 	// write out
-	for _, name := range deletedNames {
-		if prevDefault == name {
-			fmt.Printf("%s: unmarked as default\n", name)
```
Comment (priteshbandi) [inline cmd/notation/key.go:248]: We also need to show that notation removed key that was marked defualt.

Comment (Two-Hearts) [reply cmd/notation/key.go:248]: Oh yeah, based on the [PR](https://github.com/notaryproject/notation/pull/606) of the spec change, we decided to remove this message.
This change was suggested by @yizha1 and @FeynmanZhou.

Comment (yizha1) [reply cmd/notation/key.go:248]: If delete one key and it is default, then print out

`Removed default key <key_name> from Notation signing key list. The source key still exists.`

if delete multiple keys and one of the keys is default, then print out
```
Removed the following keys from Notation signing key list. The source keys still exist.
<key_name_1>
<key_name_2> (default)
<key_name_2>
```

Comment (Two-Hearts) [reply cmd/notation/key.go:248]: Had another discussion, will add the logic back. @priteshbandi @shizhMSFT

Comment (FeynmanZhou) [reply cmd/notation/key.go:248]: Agree with @priteshbandi we should remind users when it is a default key. @Two-Hearts Could you pls also update related [CLI Spec](https://github.com/notaryproject/notation/blob/main/specs/commandline/key.md#remove-two-keys-from-notation-signing-key-list) in this PR?

File: specs/commandline/key.md [hunk 2]
```diff
@@ -130,6 +130,11 @@ Removed <key_name> from Notation signing key list. The source key still exists.
 
 ```text
 Removed <key_name> from Notation signing key list. The source key still exists.
+
```
Comment (shizhMSFT) [inline specs/commandline/key.md:133]: Missing ending ` ``` `

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**